README.adoc

```diff
@@ -139,7 +139,7 @@ dynamic configuration using MicroProfile Configuration is used to allow you to e
 or the other down for maintenance. If you want to learn more about setting up dynamic configuration,
 see https://openliberty.io/guides/microprofile-config.html[Configuring microservices].
 
-To enable fault tolerance in the application, create the system client class in the
+To enable fault tolerance in the application, create the `SystemClient` class in the
```

Comment:
"create the `SystemClient` class in the" could be a more descriptive way of saying, create the system clien class in the". Don't have to use the class name. But it's ok.